recipes/jp-noui/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "0.3.4" %}
+
+package:
+  name: jp-noui
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/j/jp_noui/jp_noui-{{ version }}.tar.gz
+  sha256: 98ab266fa398311b1736a37dfcc6718bf903f7c190e94ac70f38ab5f5b57fca8
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  entry_points:
+    - "jupyter-noui = jp-noui.app:main"
+
+requirements:
+  host:
+    - hatch-jupyter-builder
+    - jupyterlab >=3,<4
+    - pip
+    - python >=3.7
+
+  run:
+    - python >=3.7
+
+  run_constrained:
+    - jupyterlab >=3,<4
+
+test:
+  requires:
+    - pip
+    - jupyterlab >=3,<4
+    - m2-grep  # [win]
+  imports:
+    - jp_noui
+  commands:
+    - pip check
+    - jupyter labextension list
+    - jupyter labextension list 1>labextensions 2>&1
+    - grep -iE "jp-noui.*v{{ version }}.*OK" labextensions
```

Comment:
for completeness on the `entry_points`, probably add:
```bash
jupyter noui --version
jupyter noui --help
```

Reply:
Yeah, that's a good idea. Thank you
00ab3a8

Reply:
Entry point was actually broken (needed to swap - with _) [a13f053](https://github.com/conda-forge/staged-recipes/pull/22083/commits/a13f0532c584b437254f1c58f6ed2a7624410b9d)